extensions/dub-link-shortener/src/search-links.tsx

```diff
@@ -226,7 +180,7 @@ const tryDeleteLink = async (linkId: string, mutate: MutatePromise<LinkSchema[]>
         primaryAction: {
           title: "Retry",
           onAction: async (toast) => {
-            await tryDeleteLink(linkId, mutate, workspaceId);
+            await tryDeleteLink(linkId, mutate);
             await toast.hide();
           },
```

Comment:
`await toast.hide()` might be a mistake on my part. I noticed this hides the toast after the first retry making second retry impossible. Can you please change this to:

```typescript
onAction: () => tryDeleteLink(linkId, mutate);
```